### PR TITLE
Update commons-io to 2.11

### PR DESCRIPTION
**Pull request by francisco-nery**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -188,6 +189,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Added to override outdated version in mule-service-http -->
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
```
Comment (giss-ignacio) [inline pom.xml:196]: scope test ?

**Review by giss-ignacio — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (francisco-nery) [reply pom.xml:196]: Provided scope you mean? The dependency appears twice, one with provided and one with test @giss-ignacio

Comment (giss-ignacio) [reply pom.xml:196]: ok. Then it's ok. But you have to add the exclusion to that mule-service-http dependency. 
```
<exclusions>
    <exclusion>
        <groupId>commons-io</groupId>
        <artifactId>commons-io</artifactId>
    </exclusion>
</exclusions>
```

Comment (giss-ignacio) [inline pom.xml:196]: you can add a property for the version

**Review by giss-ignacio — CHANGES_REQUESTED** [review]

**Review by giss-ignacio — APPROVED** [review]

**Pull request merged**